deel-3-webpagina/index.html

```diff
@@ -7,10 +7,46 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
+<header>
+    <nav>
```

Comment:
ik heb zelf geen sections gebruikt in het nav element, alleen een losse <p> voor de merknaam en een unordered list voor de linkjes rechtsboven. De bullet points in een list kun je dan weer uitzetten in CSS. Voor het resultaat maakt het niet uit, dit werkt ook.